.github/workflows/mobile-docs.yml

```diff
@@ -30,20 +30,32 @@ jobs:
     runs-on: ${{ needs.env.outputs.agent_ubuntu }}
     timeout-minutes: 20
     steps:
+    # Checkout the Envoy repo
     - uses: actions/checkout@v4
+    # Checkout the envoy-mobile/envoy-mobile.github.io repo
+    - uses: actions/checkout@v4
+      if: github.ref == 'refs/heads/main'
+      with:
+        repository: envoy-mobile/envoy-mobile.github.io
```

Comment:
are the docs not moved to the envoy repo?  i thought we're completely done with the envoy-mobile repo?

Reply:
not at this point but thats defo what i think we should

it requires a little bit more thought/discussion tho and `main` is breaking on this so probs we need to address

i just dont want to iterate this more than we have to 8/

Reply:
makes sense to get this working first, i just didn't realize docs hadn't been moved over to the envoy repo, so was puzzled by it.